tests/IntegrationTests/WorkflowExecutor.IntegrationTests/StepDefinitions/WorkflowInstanceStepDefintions.cs

```diff
@@ -79,6 +82,45 @@ public void GivenIHaveWorkflowInstances(int count)
             }
         }
 
+        [Given(@"I have (.*) failed workflow Instances with acknowledged workflow errors with mid date as (.*)")]
+        public void GivenIHaveWorkflowInstancesWithAcknowledgedWorkflowErrors(int count, string midDate)
+        {
+            _outputHelper.WriteLine($"Retrieving {count} workflow instances");
+            var listOfWorkflowInstance = new List<WorkflowInstance>();
+
+            var currentCulture = Thread.CurrentThread.CurrentCulture;
+            Thread.CurrentThread.CurrentCulture = new CultureInfo("en-GB");
```

Comment:
Do we want to test different cultures?

Reply:
Removed this reliance on CurrentCulture in integration tests but also added culture checks in unit tests.

I do you think this resolves the issue?